docs/docsource/operation/XDM_GUI/bucket_operations/delete_a_replication_workflow.rst

```diff
@@ -14,3 +14,13 @@ To delete a replication workflow rule:
 #. The workflow is removed from the menu. If no workflows remain, you are
    prompted to create a new worflow.
 
+Delete a Replication Workflow from the Command Line
+---------------------------------------------------
```

Comment:
I'd say "Deleting a Replication Workflow from the Command Line" to keep a consistency with the other two operations "Creating" & "Updating" 😉